Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -66,8 +66,8 @@ public enum ModuleError: Swift.Error {
 extension ModuleError: CustomStringConvertible {
     public var description: String {
         switch self {
-        case .duplicateModule(let name):
-            return "multiple targets named '\(name)'"
+        case .duplicateModule(let name, let packages):
+            return "multiple targets named '\(name)' in: \(packages)"
```

Comment:
Could we print them to be more human readable (like in modulesNotFound)? Something like:

```swift
let packages = packages.joined(separator: ", ")
return "multiple targets named '\(name)' in: \(packages)"
```